examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -28,21 +28,22 @@
 # a more detailed algorithm can be found in this `blog
 # <https://towardsdatascience.com/boosting-your-sequence-generation-performance-with-beam-search-language-model-decoding-74ee64de435a>`__.
 #
-# Running ASR inference using a CTC Beam Search decoder with a KenLM
-# language model and lexicon constraint requires the following components
+# Running ASR inference using a CTC Beam Search decoder with a language
+# model and lexicon constraint requires the following components
 #
 # -  Acoustic Model: model predicting phonetics from audio waveforms
 # -  Tokens: the possible predicted tokens from the acoustic model
 # -  Lexicon: mapping between possible words and their corresponding
 #    tokens sequence
-# -  KenLM: n-gram language model trained with the `KenLM
-#    library <https://kheafield.com/code/kenlm/>`__
+# -  Language Model (LM): n-gram language model trained with the `KenLM
+#    library <https://kheafield.com/code/kenlm/>`__, or custom languge
+#    model wrapped using :py:class:`~torchaudio.models.decoder.CTCDecoderLM`
```

Comment:
Nit: “wrap” sounds like a class that has the said class as attribute is also compatible. Perhaps “inherit” is unambiguous?